book/website/welcome.md

```diff
@@ -40,6 +40,16 @@ To join this community, please read our [contribution guidelines](https://github
 More information about the community and the project is available in the [Community Handbook](./community-handbook/community-handbook).
 We look forward to expanding and building _The Turing Way_ together.
 
+
+```{figure} <./figures/TheTuringWayChapters.jpg>
```

Comment:
I think you need to check the file path for this as it isn't loading in the preview - should be "figures/FILENAME.jpg" with no quote marks.

See the first image insertion in this file for an example :)

Reply:
@Arielle-Bennett have updated the code 😄